tools/gui/gui/eq_configs/eq_iir_bandpass.txt

```diff
@@ -0,0 +1 @@
+4607827,0,116,50331648,0,0,0,0,116,2,1,0,0,0,0,0,0,2,2,0,0,0,0,3316150158,2048164275,513807534,3267352229,513807534,0,16384,3867454526,1191025347,38870735,77741469,38870735,4294967292,24197,
```

Comment:
These are duplicates from tools/ctl/ip3. It would be good to align and have a single set of blobs for sof-ctl somewhere. And somehow have selection for ipc3 vs. ipc4 blobs usage. I'm also thinking to change the .txt blob headers format to be the same as .bin used by UCM2, so if duplication can be avoided there's less work to maintain.

Reply:
So, I prefer .txt over .bin especially in git where we can see the diffs. UCM2 chose a slightly different headers format for blobs and since sof-ctl is only a small utility I'm planning to change it to align with UCM2 but still keep also the .txt blobs.  Then the .bin and .txt blob content would be the same.

Reply:
Do you have any suggestions for where this should be? For now, I'll remove them from this directory and document where they are/how to add them to the gui.

Commit here: 01c19dd9101d407595b5a99978791e547075ec0a

Reply:
Could there be a configure widget in the GUI where the path could be set? It would also make this compatible with both IPC versions.

Reply:
@alexb3103  @singalsu it should be, maybe, a text box where one can write the path to their eq_configs.